packages/core/src/css/core.scss

```diff
@@ -85,9 +88,20 @@
   --way-form-element-height-medium: 3.125rem; // 50px
   --way-form-element-height-large: 4.063rem; // 65px
 
+  --way-form-element-input-border-width: 1px;
```

Comment:
**suggestion**: In mijn branch heb ik de `--way-form-element-border-width` aangepast naar 1px. Heb liever dat je dat hier ook doet en dus gewoon de `--way-form-element-border-width` gebruikt ipv nieuw aangemaakte `--way-form-element-input-border-width` (die is dan niet meer nodig).